### PR TITLE
Add Go gqlgen lib to list of federation support

### PR DESCRIPTION
**Pull request by VitaliiLakusta**
gqlgen - one of the most popular graphql Go lib - just recently added support for federation. 
PR to the lib was merged to master Jan 7 - https://github.com/99designs/gqlgen/pull/851

The lib doesn't have documentation for the feature yet, but it's fully working, as I am already using it in one of the projects.

### CODE REVIEW

File: docs/source/federation/other-servers.md [hunk 1]
```diff
@@ -12,5 +12,6 @@ The following is a list of known open source GraphQL server libraries that have
 | PHP         | wp-graphql   | [wp-graphql-federation](https://github.com/wp-graphql/wp-graphql-federation)     |
 | Python      | graphene     | [graphene-federation](https://github.com/preply/graphene-federation)             |
 | Ruby        | graphql-ruby | [apollo-federation-ruby](https://github.com/Gusto/apollo-federation-ruby)        |
+| Go          | gqlgen       | [gqlgen](https://github.com/99designs/gqlgen/tree/master/plugin/federation)      |
```
Comment (abernix) [inline docs/source/federation/other-servers.md]: Can you sort this new addition into the list alphabetically? (i.e. move it to the top)

Comment (VitaliiLakusta) [reply docs/source/federation/other-servers.md]: Done. Thanks for the comment 🙇

**Comment by trevor-scheer** [comment]
Thanks for the addition @VitaliiLakusta! Is there any mention of federation support in the gqlgen docs that I can point users to? If not, we'd be grateful for the addition!

**Comment by trevor-scheer** [comment]
Pardon my oversight, I see your commit message mentions exactly what I was asking 🙂

**Pull request merged**